<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="mainChat">
     <widget class="QFrame" name="infoFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="photoLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>10</y>
         <width>60</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="contactLabel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>420</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>ВЛАД БУБСОВ</string>
       </property>
      </widget>
      <widget class="QLabel" name="statusLabel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>50</y>
         <width>420</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Я КСТАТИ ОНЛАЙН</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chatSearchButton">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>30</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="menuButton">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>30</y>
         <width>30</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="searchLine">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="settingsButton">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>20</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>175</x>
         <y>30</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="mainFrame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>600</width>
        <height>480</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="mesFrame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>560</y>
        <width>600</width>
        <height>40</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="sticButton">
       <property name="geometry">
        <rect>
         <x>525</x>
         <y>5</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>5</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="sendLine">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>5</y>
         <width>480</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="toolWidgetM" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hideButtonM">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>0</y>
        <width>22</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButtonM">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>0</y>
        <width>22</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showButtonM">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>0</y>
        <width>22</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="signUp">
     <widget class="QLineEdit" name="logLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>400</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="conLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>400</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>320</y>
        <width>400</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="verPasswordLine">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>390</y>
        <width>400</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="conButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>470</y>
        <width>200</width>
        <height>32</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>47</green>
            <blue>48</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>47</green>
            <blue>48</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>47</green>
            <blue>48</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>47</green>
            <blue>48</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Light">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Dark">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="BrightText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>29</red>
            <green>30</green>
            <blue>59</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Shadow">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>46</red>
            <green>47</green>
            <blue>48</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>400</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Sign Up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>360</y>
        <width>171</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Confirm Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hideButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="showButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QWidget" name="toolWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>541</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
